src/matrix/room/timeline/Timeline.js

```diff
@@ -751,6 +751,8 @@ export function tests() {
             }, createEvent("m.room.message", "event_id_2", bob));
             const entryB = new EventEntry({ event });
             await timeline.load(new User(alice), "join", new NullLogItem());
+            timeline._setupEntries([]);
```

Comment:
I'd try to avoid calling private methods from tests if possible, as they can make the tests more brittle. Why not use load?